examples/example_parametrized_nodes.py

```diff
@@ -0,0 +1,110 @@
+import functools
```

Comment:
Please add more docstrings to get a nice documentation of this example in our sphinx-generated docs.